library/IvozProvider/Mapper/Sql/Brands.php

```diff
@@ -62,6 +65,26 @@ protected function _save(\IvozProvider\Model\Raw\Brands $model,
         return $pk;
     }
 
+    protected function _createDomainAttrs($model)
+    {
+        $domainName = $model->getDomainUsers();
+
+        if (!empty($domainName)) {
+            $domainAttrsMapper = new \IvozProvider\Mapper\Sql\KamUsersDomainAttrs();
+
+            $domainsAttr = $domainAttrsMapper->fetchList("did='$domainName' AND name='brandId'");
+            if (empty($domainsAttr)) {
+                $domainAttr = new \IvozProvider\Model\KamUsersDomainAttrs();
+
+                $domainAttr->setDid($domainName)
+                           ->setName('brandId')
+                           ->setType('0')
```

Comment:
It stands for 'integer' [(Kamailio doc)](http://kamailio.org/docs/modules/devel/modules/domain.html#domain.p.type_col)

This table is only used for storing the relationship between domain<->brand, needed to log messages with brand tag (e.g. [b1]) asap. 

It is not used for anything more, I don't know if adding thats constants deserves the effort...